.github/workflows/preview.yaml

```
@@ -67,9 +67,24 @@ jobs:
+          mkdir -p cypress/snapshots
+        uses: actions/download-artifact@v3
+        with:
+          name: cypress-snapshots
+          path: cypress/snapshots
+
+      - name: Run Cypress tests
         run: npm run cy:test
 
+      - name: Upload new snapshots as artifacts
+        if: always()
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)